.github/workflows/ci.yaml

```diff
@@ -9,9 +9,6 @@ jobs:
       matrix:
         ruby-version:
         - '2.7'
-        activerecord:
-        - "60"
-        - "61"
```

Comment:
don't you want to keep 6.1?

Reply:
Or wait, it is just doing the default of rails 6.1 from the Gemfile?

Reply:
but, at some point we will want to validate Rails 7.0 - so I assume we want to be able to specify multiple versions of ActiveRecord?

Reply:
we will be doing a different branch for each version

when we are ready for 7.0, then we change across.
if we need to backport to 6.1.1, we will test those changes for 6.1 only

does this plan still sound good?

Reply:
sure - just wanted to ensure it was thought through!

Reply:
@kbrock is correct...since we are moving to a "branch/release per Rails version", there's no longer a need to support multiple versions simultaneously.  The 6-1-stable branch or whatever we call it will be 6.1 only and master will be 7.0 until we cut a 7-0-stable branch.